client/Friend.js

```diff
@@ -68,4 +74,9 @@ Friend.prototype.update = function(info) {
 		if(info.position.coords.accuracy)
 			this.accuracyCircle.setRadius(info.position.coords.accuracy)
 	}
+	this.updateMarker(info.stale)
+}
+Friend.prototype.updateMarker = function(stale) {
```

Comment:
This looks like it got in here from the other pull request.

In general, a pull request should correspond to a branch off of master (in the repo you want to pull into) so that we don't bump into inter-PR dependencies.